index.rst

```diff
@@ -1,29 +1,192 @@
 :tocdepth: 1
 
-.. sectnum::
+Abstract
+========
 
-.. Metadata such as the title, authors, and description are set in metadata.yaml
+In normal operation, the Rubin Science Platform will authenticate a user via an external identity provider and ensure that all actions by that user are associated with that identity.
+However, there are cases, such as debugging user problems or dealing with certain types of security issues, where it is invaluable for an administrator to be able to access the Science Platform as if they were another user.
+They can then see exactly what that user would see and make changes as the user to fix problems.
 
-.. TODO: Delete the note below before merging new content to the main branch.
+This tech note proposes a design for implementing user impersonation in Gafaelfawr_, the authentication system for the Science Platform.
+It focuses on user impersonation to services used via the browser, such as the Notebook Aspect and the Portal Aspect.
+Impersonation of users to API services using bearer tokens is already possible by creating a new token for that user.
+This will not be changed by this proposal.
+
+.. _Gafaelfawr: https://gafaelfawr.lsst.io/
 
 .. note::
 
-   **This technote is a work-in-progress.**
+   This is part of a tech note series on identity management for the Rubin Science Platform.
+   The primary documents are DMTN-234_, which describes the high-level design; DMTN-224_, which describes the implementation; and SQR-069_, which provides a history and analysis of the decisions underlying the design and implementation.
```

Comment:
Oh, thanks! I have been using those in docs like Phalanx and Gafaelfawr, but I didn't know if they were also supported in tech notes and thus had been avoiding them just in case. I will start updating all of my tech notes to use that syntax, since I like it a lot better and it avoids occasionally having typos where I link to the wrong tech note.